GageStatsAgent/GageStatsAgent.cs

```diff
@@ -315,12 +317,21 @@ public IUser GetUserByID(int id)
         public IUser AuthenticateUser(string username, string password)
         {
             //this is where one authenticates the username/password before passing back user
-            var user = (User)GetUserByUsername(username);
-            if (user == null || !WIM.Security.Cryptography.VerifyPassword(password, user.Salt, user.Password))
+            try
             {
+                var user = (User)GetUserByUsername(username);
```

Comment:
Looking at GetUserByUsername method, looks like we need to remove `throw new NotImplementedException();`